### PR TITLE
feat: Improved accessibilty for credential cards

### PR DESCRIPTION
**Pull request by wadeking98**
# Summary of Changes

Previously:
- Difficult to select credentials from list screen
- Screen reader would read repetitive watermark
Changes:
- Made credential cards accessible as a single element in the list screen
- Changed accessibility label to be `Issued by <ISSUER_NAME>, <WATERMARK> <CREDENTIAL_NAME>`

https://github.com/hyperledger/aries-mobile-agent-react-native/assets/36937407/3253c1bd-c999-40e5-a988-d6f85816b07d

# Related Issues

N/A

# Pull Request Checklist

Tick all boxes below to demonstrate that you have completed the respective task. If the item does not apply to your this PR **check it anyway** to make it apparent that there's nothing to do.

- [x] All commits contain a DCO `Signed-off-by` line (we use the [DCO GitHub app](https://github.com/apps/dco) to enforce this);
- [x] Updated LICENSE-3RD-PARTY.md for any added dependencies or vendored components;
- [x] Updated documentation as needed for changed code and new or modified features;
- [x] Added sufficient [tests](../__tests__/) so that overall code coverage is not reduced.

If you have _any_ questions to _any_ of the points above, just **submit and ask**! This checklist is here to _help_ you, not to deter you from contributing!

Pro Tip 🤓

- Read our [contribution guide](../CONTRIBUTING.md) at least once; it will save you a few review cycles!
- Your PR will likely not be reviewed until all the above boxes are checked and all automated tests have passed.

_PR template adapted from the Python attrs project._

### CODE REVIEW

File: packages/legacy/core/App/localization/fr/index.ts [hunk 1]
```diff
@@ -260,6 +260,8 @@ const translation = {
         "CredentialsNotFound": "Justificatifs non trouvés",
         "CredentialDetails": "Détails des justificatifs",
         "EmptyList": "Votre portefeuille est vide. Vos justificatifs acceptés seront ajoutés ici.",
+        "IssuedBy": "Issued by",
```
Comment (jleach) [inline packages/legacy/core/App/localization/fr/index.ts]: Missing `(FR)`

File: packages/legacy/core/App/localization/pt-br/index.ts [hunk 2]
```diff
@@ -243,6 +243,8 @@ const translation = {
     "CredentialsNotFound": "Credenciais não encontradas",
     "CredentialDetails": "Detalhes da Credencial",
     "EmptyList": "Sua carteira está vazia. Suas credenciais aceitas serão adicionadas aqui.",
+    "IssuedBy": "Issued by",
```
Comment (jleach) [inline packages/legacy/core/App/localization/pt-br/index.ts]: Bissing `(PT-BR)`

**Review by jleach — CHANGES_REQUESTED** [review]

**Review by bryce-mcmath — COMMENTED** [review]
This is a great improvement

File: packages/legacy/core/App/localization/pt-br/index.ts (hunk 2 above)
Comment (wadeking98) [reply packages/legacy/core/App/localization/pt-br/index.ts]: done 👍

**Review by jleach — APPROVED** [review]

**Pull request merged**